tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,7 +180,7 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
```

Comment:
Agreed that we should use --version the way other commands do (like ecl.exe) rather than --build.

Where we do want interface version specified, the long intuitive name should be something like "--interface-version".. for people who haven't memorized the short easy name, something like "-iv", or if you guys have a better idea for the short name I'm open.

"--" should always be a long form very intuitive and explicit name.  So not --ver_.  Single "-" should be a short name, often an abbreviation or single letter.  But again I think -ver_ is a bit to awkward.